lib/instrumentation.ex

```diff
@@ -56,18 +59,30 @@ defmodule OpentelemetryAbsinthe.Instrumentation do
 
   def handle_operation_start(_event_name, _measurements, metadata, config) do
     params = metadata |> Map.get(:options, []) |> Keyword.get(:params, %{})
+    document = params["query"]
+    operation_name = params["operationName"]
```

Comment:
I wonder if these should be `Map.get`s, direct access makes me nervous but maybe I'm just paranoid

Reply:
Keyword.get returns nil if the map key is undefined so I think this is fine. 

too much :crab:?

Reply:
Also TIL `nil["a"] = nil` apparently, so fine I guess? :laughing: Too much JS programming left me paranoid